roll: 0
pitch: 0
yaw: 0

x_pos: 0
y_pos: 0
z_pos: .2


